RiotSwiftUI/Modules/Authentication/VerifyEmail/View/AuthenticationVerifyEmailScreen.swift

```diff
@@ -0,0 +1,151 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct AuthenticationVerifyEmailScreen: View {
+
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme
+    
+    // MARK: Public
+    
+    @ObservedObject var viewModel: AuthenticationVerifyEmailViewModel.Context
+    
+    // MARK: Views
+    
+    var body: some View {
+        GeometryReader { geometry in
+            VStack {
+                ScrollView {
+                    mainContent
+                        .frame(maxWidth: OnboardingMetrics.maxContentWidth)
+                        .frame(maxWidth: .infinity)
```

Comment:
Why is the `maxWidth` set twice here?

Reply:
This creates a view that is constrained in width for iPad, and then embeds it in a view that can take up the full width (with the default alignment of `.center`). I've noticed recently that this plays better placing scrollbar indicators on the edge of the screen compared to a constrained width on the stack and an infinite width on the scroll view.

Reply:
I see. If we will use this in more places, it might be worth extracting into a custom view modifier, which will make the intent explicit as well as allow easy change of implementation.

Reply:
This is interesting, maybe create a small extension that does this then so we don't lose this knowledge ?

Reply:
This is a nice idea 👌